### PR TITLE
Update roadmap: Q1 2023

### PR DESCRIPTION
**Pull request by sunjayBhatia**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -2,28 +2,40 @@
...
+Please take the timelines & dates as proposals and goals.
+Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc.
+If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Contour.
+
+`Last Updated: January 2023`
+
+|Theme|Description|Timeline|Issue|Notes|
+|--|--|--|--|--|
+|Gateway API|Implement GRPCRoute|2023 H1|[projectcontour/contour#4820](https://github.com/projectcontour/contour/issues/4820)|Work has started on a design document for this feature.|
+|Gateway API|Support arbitrary number of Listeners per-Gateway|2023 H1|[4960](https://github.com/projectcontour/contour/issues/4960)|Will likely be required to stay conformant with Gateway API.|
```
Comment (skriss) [inline ROADMAP.md:34]:
```suggestion
|Gateway API|Support arbitrary number of Listeners per-Gateway|2023 H1|[projectcontour/contour#4960](https://github.com/projectcontour/contour/issues/4960)|Will likely be required to stay conformant with Gateway API.|
```

File: ROADMAP.md [hunk 2]
```diff
@@ -2,28 +2,40 @@
...
+|Observability|Tracing support|2023 H1|[projectcontour/contour#399](https://github.com/projectcontour/contour/issues/399)|Work has started on a design document for this feature.|
+|Security|Add OIDC support to contour|Unknown|[projectcontour/contour#2664](https://github.com/projectcontour/contour/issues/2664)|This covers work to pull OIDC support directly into Contour, without requiring running an external auth server. We started this work, but the Oauth2 filter in Envoy is still very experimental and needs a lot of work. This work can’t proceed until significant work has been done on the Oauth2 filter.|
```
Comment (skriss) [inline ROADMAP.md:36]: Let's move this down to just above the "Won't Do" UDPRoute item since its timeline is unknown.

**Review by skriss — CHANGES_REQUESTED** [review]
Thanks for this update @sunjayBhatia, looks great. Couple minor comments.

File: ROADMAP.md (hunk 1 above)
Comment (sunjayBhatia) [reply ROADMAP.md:34]: good catch

File: ROADMAP.md (hunk 2 above)
Comment (sunjayBhatia) [reply ROADMAP.md:36]: 👍🏽

File: ROADMAP.md [hunk 3]
```diff
@@ -2,28 +2,40 @@
...
+|Security|IP Block list|2023 H1|[projectcontour/contour#2971](https://github.com/projectcontour/contour/issues/2971) and [projectcontour/contour#3693](https://github.com/projectcontour/contour/issues/3693)|Work has started on a design document for this feature.|
+|Infrastructure|Replace Contour xDS code with go-control-plane as default|2023 H2|[projectcontour/contour#2134](https://github.com/projectcontour/contour/issues/2134)|There’s some work left to do, but this should be finished soon.These changes will also make Delta xDS available, which should help with wire transfer times and scalability.|
```
Comment (sunjayBhatia) [inline ROADMAP.md:38]: my bad on this, need to pick this up again, possibly a good way to help with some of the upgrade flakiness and other issues

Comment (skriss) [reply ROADMAP.md:37]: Yeah this would be a nice one to get done!

**Review by skriss — APPROVED** [review]

**Review by tsaarni — APPROVED** [review]
Looks good to me! :+1:

**Pull request merged**